docker/entrypoint.sh

```diff
@@ -1,3 +1,21 @@
 #!/bin/bash
+
+SSH_PORT=${SSH_PORT:-1122}
+
+echo "Port 1122" >> /etc/ssh/sshd_config
```

Comment:
File /etc/ssh/sshd_config  may grow up with a lot of "Port 1122" after multiple restart ?